demo-app/build.gradle.kts

```diff
@@ -58,28 +58,30 @@ android {
 }
 
 dependencies {
-    implementation(libs.androidx.appcompat)
-    implementation(libs.androidx.constraintlayout)
-    implementation(libs.material)
-    implementation(libs.androidx.lifecycle.livedata.ktx)
-    implementation(libs.androidx.lifecycle.viewmodel.ktx)
-    implementation(libs.androidx.navigation.fragment.ktx)
-    implementation(libs.androidx.navigation.ui.ktx)
-    coreLibraryDesugaring(libs.desugarJdkLibs)
+    implementation(demoLibs.androidx.appcompat)
```

Comment:
I think there's no need to change the prefix to `demoLibs` after doing what I mentioned in this [other comment](https://github.com/open-telemetry/opentelemetry-android/pull/365/files#r1602909961).